internal/rawhelpers.go

```diff
@@ -20,7 +20,10 @@ func RawToBool(r uint64) bool {
 }
 
 func Int64ToRaw(i int64) uint64 {
-	return uint64(i)
+	if i < 0 {
+		return 0
+	}
```

Comment:
This will break `attribute.Int64Value` for negative numbers.
We should add a unit test for it to avoid a regression bug.
See also `RawToInt64` implementation for reference.

Reply:
Wouldn't we already have an overflow exception for negative numbers?
I'll add a test, but this doesn't feel like a regression. We're removing a fatal.